frontend/lib/laletterbuilder/letter-builder/choose-letter.tsx

```diff
@@ -18,20 +21,126 @@ export const LaLetterBuilderChooseLetterStep: React.FC<ProgressStepProps> = (
       className="content"
       withHeading="small"
     >
+      <LetterCard
+        title="Notice to Repair"
+        time_mins={15}
+        text={`Document repairs needed in your home, and send a formal request to your landlord`}
+        buttonProps={{
+          to: LaLetterBuilderRouteInfo.locale.habitability.latestStep,
+          className: "button is-primary is-medium",
+          text: li18n._(t`Start a letter`),
+        }}
+        information={"notice to repair information needed"}
+      />
+      <LetterCard
+        title="Right to Privacy"
+        time_mins={15}
+        text={`Your landlord can't enter your unit whenever they want. Create a formal request 
+        which asks your landlord to follow proper protocol and respect your right to privacy.`}
+        buttonProps={{
```

Comment:
Does this text need to be internationalized in any way?

Reply:
done